### DISCUSSION TITLE
Animation

### DISCUSSION THREAD

**Discussion by mike-lloyd03** [Schweber/hdrop#8] [General]
Thanks for this script. I used to use scratchpads for a dropdown terminal but it wasn't ideal. This works better but I miss having a nice `slidefadevert` animation when the terminal would drop in. I tried adding:

```hypr
windowrulev2 = animation slidefadevert,class:^(dropdown_terminal)$
```

But that animation only runs when the window is first opened, not when it's moved back into the active workspace. Any ideas?

**Comment by Schweber** [comment]
I noticed this as well while working on the `--floating` option. I had added an option to choose an animation but it only took effect the first time the window is opened so i removed it again. Sadly, i have no workaround.

I think it would be worth it to open an issue with hyprland, because this behaviour might just be an oversight. As hdrop uses hyprland in a way that most users don't, it is prone to stumble upon such issues.
